cli/proc_state.rs

```diff
@@ -567,7 +570,7 @@ impl ProcState {
     let referrer = if referrer.is_empty() && is_repl {
       deno_core::resolve_path("./$deno$repl.ts", self.options.initial_cwd())?
```

Comment:
Actually, I'm not sure this one should be initial cwd because someone in the repl might be changing directories and want to resolve a module based on their new cwd?